printer/colors_definition.go

```diff
@@ -38,3 +38,51 @@ var (
 	HeaderColorForLight   = color.Black // for plain table
 	RequiredColorForLight = color.Red   // for `kubectl explain`
```

Comment:
Can these color settings be removed then?

Reply:
yep, a lot of cleanup...